### PR TITLE
Project : Using Bootstrap

### PR DESCRIPTION
**Pull request by LamiaSristy**
### Newsweek.com clone.

- Here we used Bootstrap CSS classes 
- We have also used some set of CSS classes
- Used HTML

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
-# NewsWeek-Copy
\ No newline at end of file
+# NewsWeek-Copy
```
Comment (lmaldonadoch) [inline README.md]: - [x] [OPTIONAL] Change Copy to Clone, it has a more positive and professional denotation.

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+> In this project we made a replica of the news site Newsweek.com using the Bootstrap framework.
+
+- Laptop view
+![screenshot](img/screencapture.png)
+
+- mobile view
+![screenshot-mobile](img/mobile-screenshot.jpeg)
+
+- ipad view
+![screenshot-tab-view](img/ipad-screencaptur.png)
```
Comment (lmaldonadoch) [inline README.md]: - [x] [OPTIONAL] The images are way too big. They should be only a placeholder to illustrate a little bit of the project. The most important here are the authors. What you learned, what challenges you overcame. Try to elaborate on that, it gives anybody who visits your repo a good feeling of who created this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1135 @@
...
+                            </figure>
+                            <a href="">
+                                The Facebook Ad Boycott Has Failed, and Mark Zuckerberg Is Richer Than Ever
+                            </a>
+                            <p>
+                                Mark Zuckerberg's net worth exceeded $100 billion just days after an ad boycott targeting Facebook ended.
+                            </p>                
+                        </div> 
+                    </div>          -->
+                <!-- </div> -->
```
Comment (lmaldonadoch) [inline index.html]: - [x] [REQUIRED] Remove commented code. If you ever need to go back and check some code from the past, use your version control tools (GitHub).

**Review by lmaldonadoch — CHANGES_REQUESTED** [review]
# Changes Required 🥇  🏃‍♂️ 

## Great job, team!!! 🏆 

You blew me away with your first review! Your project looks very similar to the original one! There are some changes that we need to work on before getting approved, but with your talent, I'm sure it'll be ready in no time!

- [x] [REQUIRED] The logo on the navbar should be a link.

- [x] [REQUIRED] The subscribe link should have the right arrow.
image

- [x] [REQUIRED] The nav links should not wrap.
image

- [x] [REQUIRED] Your container max-width is not the same and is not responsive. On medium size, the content should occupy 100% of the screen width.
image
 
- [x] [REQUIRED] On mobile devices, your Top Story should come before the Featured Story. This is challenging. Hint 💡 do the same that you did with the navbar (d-none).
image

- [x] [REQUIRED] The OPINION title is wrapping down on the small and medium-size screens.
image

- [x] [REQUIRED] On mobile devices More Stories section should come before My Turn, and My Turn should be displayed as a col-12 each article.
image

- [x] [REQUIRED] Double-check your page layout. As mentioned before, the max-width of your container seems to be a little off. Remember the devil is in the details, and it is the attention to detail that differentiates a good developer from a GREAT developer.
image



Seems like a lot! <strong>I know!!!</strong> But it is mostly small details, you did an <strong>AMAZING</strong> job, your implementation of Bootstrap along the whole page is breathtaking, you barely used any CSS. That is something you should feel very proud of! 👏 

Please implement the required changes, and think about the optional ones.

It was a pleasure reviewing your code,

Happy coding! 💻 

Slack: Luis Angel Maldoando
[LinkedIn](https://www.linkedin.com/in/lmaldonadoch/)

File: README.md (hunk 1 above)
Comment (LamiaSristy) [reply README.md]: Changed in README.md file.

File: README.md (hunk 2 above)
Comment (LamiaSristy) [reply README.md]: Removed the mobile and iPad view screenshots.

File: index.html (hunk 3 above)
Comment (LamiaSristy) [reply index.html]: Removed the commented section.

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @LamiaSristy and @tiagoes :wave: 

## Congratulations, your project is approved :clap: :clap: :clap: 

## I have no words, this project is amazing!

impressive

## I have only one **OPTIONAL** challenge for you:

- [ ] **(OPTIONAL)** In the original page, the three main columns are aligned both at the top and at the bottom, try making this work:

## Yours:

Screenshot from 2020-08-14 16-05-13

## Original:

Screenshot from 2020-08-14 16-06-06

## Anyway, you may merge when you are ready! :grin: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**